seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1376,17 +1376,17 @@ def test_load_phenotype_prioritization_data(self, mock_subprocess, mock_send_ema
                                           nested_fields=[{'fields': ('individual', 'guid'), 'key': 'individualGuid'}])
         self.assertListEqual(saved_data, EXPECTED_UPDATED_LIRICAL_DATA)
         self._assert_expected_notifications(mock_send_email, [
-            {'data_type': 'Lirical', 'user': self.data_manager_user, 'email_body': 'Lirical data for 2 sample(s)'},
-        ], has_html=True)
+            {'data_type': 'Lirical', 'user': self.data_manager_user, 'email_body': 'data for 2 Lirical sample(s)'},
+        ])
 
     @staticmethod
-    def _assert_expected_notifications(mock_send_email, expected_notifs: list[dict], has_html=False):
+    def _assert_expected_notifications(mock_send_email, expected_notifs: list[dict]):
         calls = []
         for notif_dict in expected_notifs:
             project_guid = notif_dict.get('project_guid', PROJECT_GUID)
             project_name = notif_dict.get('project_name', '1kg project nåme with uniçøde')
             url = f'https://test-seqr.org/project/{project_guid}/project_page'
-            project_link = f'<a href={url}>{project_name}</a>' if has_html else f'<{url}|{project_name}>'
+            project_link = f'<a href={url}>{project_name}</a>'
```

Comment:
this test change reflects the actual bug fix - we should never be sending an email with the URL formatted in anything other than html